gapis/replay/interfaces.go

```diff
@@ -33,7 +33,7 @@ type Support interface {
 
 	// CanReplayOn returns true if the API can be replayed on the specified
 	// device.
```

Comment:
The MemoryLayout represents the MemoryLayout that was contained in the Architecture Atom at the start of the capture.

We cannot use the device object in the capture, because it is null at this point, for all traces. IIRC it gets filled in later when we go and actually start the replay, but at this point all we have is the MemoryLayout. This should all change once we get a header that contains this information.